src/JitInterface/src/ThunkGenerator/Program.cs

```diff
@@ -302,6 +411,103 @@ protected IntPtr CreateUnmanagedInstance()
 ");
         }
 
+        static void WriteNativeWrapperInterface(TextWriter tw, IEnumerable<FunctionDecl> functionData)
+        {
+            tw.Write(@"
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+// DO NOT EDIT THIS FILE! It IS AUTOGENERATED
+#include <stdarg.h>
+#include <stdlib.h>
+#include ""corinfoexception.h""
+
+struct CORINFO_LOOKUP_KIND;
```

Comment:
A basic declaration of the type still appears needed, since I need to include the prototype of getLocationOfThisType method in JitInterfaceWrapper to take its space in vtable, with call signature matching what JIT expects.